trace/gRPC.md

```diff
@@ -0,0 +1,55 @@
+# gRPC Trace
+
+This document explains tracing of gRPC requests with OpenCensus.
+
+## Spans
+
+Implementations MUST create a span, when the gRPC call starts, for the client and a span for the 
+server.
+
+Span name is formatted as (also known as full gRPC method name):
+
+* $package.$service/$method
+
+Examples of span names:
+
+* grpc.test.EchoService/Echo
+
+Outgoing requests should be a span kind of CLIENT and
+incoming requests should be a span kind of SERVER.
+
+For backends that does not support Span.Kind, the exported span names can be prefixed with `Sent.`
+(for CLIENT kind) and `Recv.` (for SERVER kind).
+
+## Propagation
+
+Propagation is how SpanContext is transmitted on the wire in an gRPC request.
+
+The propagation MUST inject a SpanContext (as a gRPC metadata `grpc-trace-bin`) and MUST extract 
+a SpanContext from the gRPC metadata. The serialization format is defined
+[here](../encodings/BinaryEncoding.md).
+
+## Status
+
+Implementations MUST set status which should be the same as the gRPC client/server status. The 
+mapping between gRPC canonical codes and OpenCensus status codes can be found
+[here](https://github.com/grpc/grpc-go/blob/master/codes/codes.go).
```

Comment:
nit: use this link instead: https://github.com/grpc/grpc/blob/master/doc/statuscodes.md?